app/kumactl/cmd/root.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/Kong/kuma/pkg/cmd/version"
 	"github.com/Kong/kuma/pkg/core"
 	kuma_log "github.com/Kong/kuma/pkg/log"
+	kuma_version "github.com/Kong/kuma/pkg/version"
+)
+
+var (
+	kumactlLog = core.Log.WithName("kumactl")
```

Comment:
Hmm, do we want that? Shall we just go with the way other commands log their errors?